### PR TITLE
nye metoder og variabler

### PR DESCRIPTION
**Pull request by Shadeschou**
Tilføjet metoden ErEmilEnGed og variablen emilErEnGed

### CODE REVIEW

File: Training.cs [hunk 1]
```diff
@@ -3,8 +3,13 @@
 public class Training
 {
     int i = 0;
-
+    bool emilErEnGed;
+	
 	public Training()
 	{
 	}
+    public erEmilEnGed()
```
Comment (LimeIcing) [inline Training.cs]: PascalCase in method names.

File: Training.cs [hunk 2]
```diff
@@ -3,8 +3,13 @@
 public class Training
 {
     int i = 0;
-
+    bool emilErEnGed;
```
Comment (LimeIcing) [inline Training.cs]: Booleans should be worded as a question.

File: Training.cs (hunk 1 above)
Comment (Shadeschou) [reply Training.cs]: I know,wanted to test your reviewing skills

File: Training.cs (hunk 2 above)
Comment (Shadeschou) [reply Training.cs]: is it really a question though

**Pull request merged**